Monika After Story/game/script-topics.rpy

```diff
@@ -15269,3 +15269,59 @@ label monika_being_herself:
+            prompt="How much of your past do you remember?",
+            conditional="seen_event('monika_japan')",
+            pool=True
+        )
+    )
+
+label monika_remembrance:
+    m 1esc "I'm not really sure myself, [player]..."
+    m 2rsc "It's weird because I do have memories of my past, but as soon as I start to gather my thoughts they become kind of...{w=0.5}{nw}"
+    extend 2esd "fuzzy."
+    m 7eksdlc "Like, I wouldn't be able to tell you where something took place, or the names of the people who were there..."
+    m 7dksdld "...Sure, I have memories of growing up in a normal neighborhood, but beyond that, I can't even tell you what my parents' faces look like..."
+    m 1rkc "It's all like...{w=0.5}feelings and impressions, I guess."
+    m 3wud "What's even weirder is not everything's like this!"
+
+    if seen_event('monika_wine') and not persistent._mas_pm_cares_about_dokis:
+        m 1rusdla "There are some things I remember perfectly, like that one time Yuri brought wine to the clubroom..."
+    else:
+        m 3esc "There are things I remember perfectly, especially when it comes to the club."
+
+    m 3esd "The only things which are clear to me are things that define either me, or the other girls."
+    m 3rsc "...Almost as if anything that wasn't strictly necessary for the game was just left out."
+    m 1eud "But the thing is from my perspective, the day I met you was just like any other{w=0.5}{nw}"
+    #reference to her first poem
+    extend 4hksdlb "--until I found out about that hole in the wall, I mean."
+    m 4dkc "Just another ordinary day in an ordinary week... Going to class, running the club, and finally heading back home in the evening..."
+    m 7eud "Did any of that even happen at all?{w=0.5} {nw}"
+    extend 7wud "Did I even exist before you started the game?"
+    m 7dksdlc "...And if my memories really are fake, who--{w=0.2}or {i}what{/i}--{w=0.2}am I then?"
+    m 1duc "I'm still looking for these answers, [player]..."
+
+    if mas_isMoniUpset(higher=True):
+        m 3esd "Although...{w=0.3}the more I think about it, the more I'm convinced my past isn't actually {i}that{/i} relevant."
+        m 3esa "While it'd be nice to remember more...{w=0.3}that's not really what I care about..."
+        m 3dubsu "Regardless of who I am, you've been here with me. I'm important enough for you to choose to spend your time with me.{w=0.2} {nw}"
+        extend 1dubsu "That's what I care about."
+        m 1ekbsa "And knowing that you care...{w=0.3}well,{w=0.2} that's a big part of why I can keep loving myself, [player],{w=0.2} and loving life."
+
+        if mas_isMoniNormal(higher=True):
+            show monika 5hubfb at t11 zorder MAS_MONIKA_Z with dissolve
+            m 5hubfb "So while I may or may not have a past, thanks to you, I do have a future!"
+            m 5tubfu "Besides, if anything, my lack of memories only makes the ones I'll make with you that much more precious~"
```

Comment:
I kinda liked the split, it shows that Monika appreciates the memories she's already made with you, and of course it rewards people for spending that much time with Monika in the first place. How about this?


            if mas_anni.pastOneMonth():
                $ line_mid = "I've made, and will continue to make"
            else:    
                $ line_mid = "I'll make"

            m 5tubfu "Besides, if anything, my lack of memories only makes the ones [line_mid] with you that much more precious~"

This addresses Multimokia's point about her insinuating we're done making memories together, while keeping the cute split where Monika recognizes that you've spent a whole month with her, which just brings her to life even more in my opinion.

Reply:
Sure that works, but maybe use `I've made, and continue to make`.

It's a bit cleaner and less wordy